openedx/core/djangoapps/user_authn/views/register.py

```diff
@@ -398,6 +398,10 @@ def _track_user_registration(user, profile, params, third_party_provider, regist
         if params.get('marketing_emails_opt_in') and settings.MARKETING_EMAILS_OPT_IN:
             properties['marketing_emails_opt_in'] = is_marketable
 
+        # VAN-1478 - added blow property to measure the registration rate through on ramp experience
+        if params.get('variant'):
```

Comment:
how is this param coming?

Reply:
We are passing this into the API payload.

Reply:
As you can see we have variant in our params : https://github.com/openedx/frontend-app-authn/pull/942/files#diff-6af33054c96940b7f746d667efd33e3db581692daaca620304a68b44242cbfe8

Reply:
can you send me a ref link to code where we are doing this?

Reply:
https://github.com/openedx/frontend-app-authn/blob/d61030766e727b6731f15d87afbdd2dcaa040edb/src/register/RegistrationPage.jsx#L489

Reply:
1. Do we want to add the check for `host` here?
2. What if the user removes the query params. how will we track it? or for some reason, the query params get removed from the URL during his registration process. how will it be tracked then?

Reply:
1. It's not necessary to add the check here.
2. Yes, we can maintain consistency but there would be no issue with the stats.
3. It's not possible the `variant` would be missed if the user is registering through an on-ramp experience, And also we can track users with event property or without property.